BuildResidentialHPXML/measure.rb

```diff
@@ -4006,6 +4004,14 @@ def self.set_building_construction(hpxml_bldg, args)
     end
   end
 
+  def self.set_schedules(hpxml_bldg, args)
+    if args[:schedules_filepaths].is_initialized
+      schedules_filepaths = args[:schedules_filepaths].get.split(',').map(&:strip)
+    end
+
+    hpxml_bldg.header.schedules_filepaths = schedules_filepaths
```

Comment:
Set the array of schedule filepaths on the building "header" instead of the top level "header.